Update README.md to add snapshot action generation entry (#399)
Telefonica/mistica-android: README.md

- Update the version of mistica to the one the snapshot was released with, sync your project and the dependency should be resolved with the artifact including your local changes.

## Snapshot generation with GitHub Action

> To create a new snapshot go to [the snapshot.yml Action](https://github.com/Telefonica/mistica-android/actions/workflows/snapshot.yml), select a branch and enter the version, like

Reviewer: Perhaps we could also add the text: press `Run workflow` before selecting a branch.

> ``` x.x.x-SNAPSHOT ```

Reviewer 2: after having reviewed the code, you don't need to write "SNAPSHOT". That suffix is added later on https://github.com/Telefonica/mistica-android/blob/0fc068f1269dfbcbb4188774b0d5d2a23e9f16b3/build.gradle#L46

I would specify that. The version must be entereed without the snapshot suffix

Author: Done

> After a while, the new version will be available!

Reviewer 2: I would still mention "SNAPSHOT" suffix somewhere. What I meant in my previous comment was that setting the suffix was not required by the workflow. However I would still specify that the release is published with the mentioned SNAPSHOT suffix. Sorry for the misunderstanding

Author: Done

> This is another option to generate a snapshot version available for all applications. Just navigate to [the snapshot.yml workflow within Actions](https://github.com/Telefonica/mistica-android/actions/workflows/snapshot.yml), select a branch and enter the version like

Reviewer: I would summarise more the text
```suggestion
In order to create a new snapshot navigate to [the snapshot.yml Action](https://github.com/Telefonica/mistica-android/actions/workflows/snapshot.yml), select a branch and enter the version like
```

> ``` x.x.x-snapshot ```

Author: Done!
